### PR TITLE
Save range in localStorage

### PR DESCRIPTION
**Pull request by dantehemerson**
* Create `use-local-state` hook 🎉

### CODE REVIEW

File: src/map_science/core.cljs [hunk 1]
```diff
@@ -70,10 +70,15 @@
                          :width 5
                          :background-color (if isDragged "#548BF4" "#CCC")}})))
 
-
+;; Creating own React hooks
+(defn use-local-state [tkey initial]
+  (let [[v, u] (hooks/use-state (if (js/localStorage.getItem tkey) (js/JSON.parse (js/localStorage.getItem tkey)) initial))]
```
Comment (dantehemerson) [inline src/map_science/core.cljs]: @f1729  I'm looking for a way to simplify it. Do you know any?

Comment (f1729) [reply src/map_science/core.cljs]: Yes, first, the function that verify if exist some value in localStorage should be in another function, functional programming promote the use of little functions that do little tasks.

Comment (f1729) [reply src/map_science/core.cljs]: You could be more descriptive in -> let [[v, u]

Comment (f1729) [reply src/map_science/core.cljs]: The rest LGTM

Thank you! I will add you in the acknowledgement :+1:

**Review by f1729 — COMMENTED** [review]
Please, use
https://github.com/bbatsov/clojure-style-guide#lisp-case

**Review by f1729 — CHANGES_REQUESTED** [review]
Check it
https://github.com/Lokeh/helix-todo-mvc/blob/17f8e2f7cc52897e21d42b8880930382618f48e3/src/todo_mvc/utils.cljs

**Comment by f1729** [comment]
Please check https://github.com/bbatsov/clojure-style-guide#bindings-alignment

File: src/map_science/core.cljs [hunk 2]
```diff
@@ -69,13 +69,21 @@
...
 
 ;; Creating own React hooks
-(defn use-local-state [tkey initial]
-  (let [[v, u] (hooks/use-state (if (js/localStorage.getItem tkey) (js/JSON.parse (js/localStorage.getItem tkey)) initial))]
-    (hooks/use-effect [v]
-      (js/localStorage.setItem tkey (js/JSON.stringify v)))
-    [v u]))
+(defn use-local-state [key initialValue]
+  (let [[state, set-state] 
+    (hooks/use-state
```
Comment (f1729) [inline src/map_science/core.cljs:82]: Alignment binding https://github.com/bbatsov/clojure-style-guide#bindings-alignment

**Review by f1729 — APPROVED** [review]

**Pull request merged**